graphql_compiler/ast_manipulation.py

```diff
@@ -45,7 +57,9 @@ def safe_parse_graphql(graphql_string):
     return ast
 
 
-def get_only_query_definition(document_ast, desired_error_type):
+def get_only_query_definition(
+    document_ast: DocumentNode, desired_error_type: Type[Exception]
+) -> OperationDefinitionNode:
     """Assert that the Document AST contains only a single definition for a query, and return it."""
     if not isinstance(document_ast, DocumentNode) or not document_ast.definitions:
```

Comment:
Now that there's a type hint, do we still need to perform the `isinstance` check?

Reply:
Unfortunately, yes — until all uses of this function are typed to something stricter than `Any`, the `isinstance` check is still meaningful and can affect code execution.